Snake/Snake/UILayer.cs

```diff
@@ -12,14 +13,17 @@ class Program
         private static BusinessLayer.gameMode currentGameMode;
         static bool gameSelected = false;
 
+        enum mazeLevel{ easy = 1, medium, hard}

```

Comment:
If you pull this enum into its own file you won't need to duplicate it like you're doing here